test/functional/loopTestDriver.py

```diff
@@ -18,6 +18,10 @@ class LoopTestDriver():
     def setUp(self, marionette):
         self.marionette = marionette
 
+    def set_context(self, context):
+        self.context = context
+        self.marionette.set_context(context)
```

Comment:
Once we sort out the e10s situation, I believe this won't be necessary, so I'm not going to file a bug for that at the moment. I will add a comment.